specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1568 @@
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "alertsMetaData": {
+      "type": "object",
+      "description": "alert meta data information.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        }
+      }
+    },
+    "alertsMetaDataProperties": {
+      "type": "object",
+      "description": "alert meta data property bag",
+      "discriminator": "metadataIdentifier",
+      "required": [
+        "metadataIdentifier"
+      ],
+      "properties": {
+        "metadataIdentifier": {
+          "type": "string",
+          "description": "Identification of the information to be retrieved by API call",
+          "enum": [
+            "MonitorServiceList"
+          ],
+          "x-ms-enum": {
+            "name": "metadataIdentifier",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "MonitorServiceList": {
+      "type": "object",
+      "title": "Monitor service details",
+      "description": "Monitor service details",
+      "required": [
+        "data"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        },
+        {
+          "type": "object"
+        }
+      ],
+      "properties": {
+        "data": {
+          "description": "Array of operations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitorServiceDetails"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "MonitorServiceDetails": {
+      "type": "object",
+      "description": "Details of a monitor service",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Monitor service name"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Monitor service display name"
+        }
+      }
+    },
+    "alertEnrichmentResponse": {
+      "description": "The alert's enrichments.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/alertEnrichmentProperties"
+        }
+      }
+    },
+    "alertEnrichmentsList": {
+      "description": "List the alert's enrichments.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List the alert's enrichments",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/alertEnrichmentResponse"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page."
+        }
+      }
+    },
+    "alertEnrichmentProperties": {
+      "type": "object",
+      "description": "Properties of the alert enrichment item.",
+      "properties": {
+        "alertId": {
```

Comment:
>alertId [](http://example.com/codeflow?start=9&length=7)

Include information in the description about what kind of ID this is (e.g. resource ID, integer, guid, etc).

Reply:
Added GUID.